<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapshotGraphDialog</class>
 <widget class="QDialog" name="SnapshotGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="currentRuntimeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Run Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="testStartTimeDisplay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="startAndEndTimeGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>631</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="startTimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="snapshotStartTimeEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="snapshotEndTimeEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="endTimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Time:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="startAndEndTimeUnitComboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>73</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hr</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="endTimeUnitLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="pastTimeGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="goBackLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Go Back:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="goBackUnitComboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>73</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hr</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="snapshotGoBackTimeEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="testStartTimeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Test Start Time: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentRuntimeDisplay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="graphCustomStartEndRadioBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Graph Custom Start/End Times</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="graphPastTimeRadioBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Graph Past x Amount of Time</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dummyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>(Hidden)</string>
   </property>
  </widget>
  <zorder>startAndEndTimeGroupBox</zorder>
  <zorder>currentRuntimeLabel</zorder>
  <zorder>testStartTimeDisplay</zorder>
  <zorder>pastTimeGroupBox</zorder>
  <zorder>testStartTimeLabel</zorder>
  <zorder>currentRuntimeDisplay</zorder>
  <zorder>graphCustomStartEndRadioBtn</zorder>
  <zorder>graphPastTimeRadioBtn</zorder>
  <zorder>okButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>dummyButton</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
